tests/std/tests/impure_matrix.lst

```diff
@@ -7,7 +7,7 @@ PM_CL="/w14640 /Zc:threadSafeInit-"
 RUNALL_CROSSLIST
 PM_CL="/EHsc /MD /D_ITERATOR_DEBUG_LEVEL=0 /std:c++14"
 PM_CL="/EHsc /MD /D_ITERATOR_DEBUG_LEVEL=0 /std:c++14 /D_HAS_IF_CONSTEXPR=0"
-PM_CL="/EHsc /MD /D_ITERATOR_DEBUG_LEVEL=0 /std:c++17"
+PM_CL="/EHsc /MD /D_ITERATOR_DEBUG_LEVEL=0 /std:c++17 /Zc:noexceptTypes-"

```

Comment:
Why is a C++17 configuration being altered here, when C++20 configurations are being altered elsewhere? (The intention was for the impure matrix to be exactly like the usual matrix, minus The Eternal Nemesis.)

Reply:
I fixed them to follow that intent.